smartseq2_single_sample/run_multi_samples/ss2_single_sample_wf.wdl

```diff
@@ -0,0 +1,300 @@
+    File dedup_bamfile = "${output_filename}.MarkDuplicated.bam"
+  }
+  runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.3.1-1504795437"
+    memory: "4 GB"
+    disks: "local-disk 20 HDD"
+  }
+}
+
+task CollectInsertMetrics {
+  File aligned_bam
+  String output_filename
+
+  command {
+    java -Xmx4g -jar /usr/gitc/picard.jar CollectInsertSizeMetrics \
+      INPUT=${aligned_bam} \
+      OUTPUT="${output_filename}.insert_size_metrics" \
+      HISTOGRAM_FILE="${output_filename}.insert_size_histogram.pdf" \
+  }
+  output {
+    File insert_metrics = "${output_filename}.insert_size_metrics"
+    File histogram = "${output_filename}.insert_size_histogram.pdf"
+  }
+  runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.3.1-1504795437"
+    memory: "4 GB"
+    disks: "local-disk 10 HDD"
+  }
+}
+
+workflow Ss2RunSingleSample {
+  File fastq_read1
+  File fastq_read2
+  File gtf
+  File ref_fasta
+  File rrna_interval
+  File ref_flat
+  String star_genome
+  String rsem_genome
+  String output_prefix
+  
+  call Star {
+    input:
+      input_fastq_read1 = fastq_read1,
+      input_fastq_read2 = fastq_read2,
+      gtf = gtf,
+      star_genome = star_genome,
+      sample_tag = output_prefix,
+      pu_tag = output_prefix,
+      lib_tag =output_prefix,
+      id_tag = output_prefix
+  }
+ 
+  call RsemExpression {
+    input:
+      trans_aligned_bam = Star.output_bam_trans,
+      rsem_genome = rsem_genome,
+      rsem_out = output_prefix
+  }
+
+  call FeatureCountsUniqueMapping {
+    input:
+      aligned_bam = Star.output_bam,
+      gtf = gtf,
+      fc_out = output_prefix
+  }
+  
+  call FeatureCountsMultiMapping {
+    input:
+      aligned_bam=Star.output_bam,
+      gtf = gtf,
+      fc_out = output_prefix
+  }
+
+  
+  call CollectRnaSeqMetrics {
+    input:
+      aligned_bam = Star.output_bam,
+      ref_genome_fasta = ref_fasta,
+      rrna_interval = rrna_interval,
+      output_filename = "${output_prefix}",
+      ref_flat= ref_flat
+  }
+
+  call CollectAlignmentSummaryMetrics {
+    input:
+      aligned_bam = Star.output_bam,
+      ref_genome_fasta = ref_fasta,
+      output_filename = "${output_prefix}"
+  }
+  
+  call CollectDuplicationMetrics {
+    input:
+      aligned_bam = Star.output_bam,
+      output_filename = "${output_prefix}"
+  }
+
+  call CollectInsertMetrics {
+    input:
+      aligned_bam = Star.output_bam,
+      output_filename = "${output_prefix}"
+    }
+  
+
+  output {
+    File bam_file = Star.output_bam
+    File bam_trans = Star.output_bam_trans
+    File rna_metrics = CollectRnaSeqMetrics.rna_metrics
+    File aln_metrics = CollectAlignmentSummaryMetrics.alignment_metrics
+    File dedup_metrics = CollectDuplicationMetrics.dedup_metrics
+    File dedup_bam = CollectDuplicationMetrics.dedup_bamfile
+    File insert_metrics = CollectInsertMetrics.insert_metrics
+    File insert_Histogram = CollectInsertMetrics.histogram
```

Comment:
`insert_histogram` would be more consistent with the format of the other output variable names

Reply:
changed.